CONTRIBUTING.md

```diff
@@ -58,6 +58,23 @@ $ tox -e fuzz
 $ black-primer [-k -w /tmp/black_test_repos]
 ```
 
+### News / Changelog Requirement
+
+`Black` has CI that will check for an entry corresponding to your PR in `CHANGES.md`. If
+you feel this PR not require a `CHANGES.md` please state that in a comment and a
+maintainer can add a `skip_news` tag to make the CI pass. Otherwise, please ensure you
+have a line in the following format:
+
+```md
+- `Black` is now more awesome (#X)
```

Comment:
watch out the regex will kill you without the PR prefix :D

```suggestion
- `Black` is now more awesome (PR #X)
```

Reply:
> watch out the regex will kill you without the PR prefix :D

Should we fix the regex - My bad, seems there are no 'PR' used in CHANGES.md ... I screwed up there it seems ...